docs/content/comparison-to-other-systems.md

```diff
@@ -265,31 +265,35 @@ denied, it is always denied.
 ```
 
 In OPA, you write each of the AWS `allow` statements as a separate statement, and you
-expect the input to have `user`, `action`, and `resource` fields.
+expect the input to have `principal`, `action`, and `resource` fields.
 
 ```ruby
 package aws
 
 # input = {
-#     "user": "alice",
-#     "action": "ec2:StartInstance"
+#     "principal": "alice",
+#     "action": "ec2:StartInstance",
 #     "resource": "arn:aws:ec2:::instance/i78999879"
 # }
+default allow = false
 
 # FirstStatement
 allow {
+    input.principal == "alice"
```

Comment:
We might want to leave out the `input.principal` check in the `allow` rules since the example above it doesn't have any mention of that. As I understand it this should be showing an equivalent OPA policy, but with this change it is doing more.

Reply:
Thanks for reminding. I changed the example a bit. Since in AWS, this kind of policy is only useful when it is attached to a principal, I change it to be attached to Alice.